consul/datadog_checks/consul/consul.py

```diff
@@ -93,13 +98,11 @@ def consul_request(self, instance, endpoint):
 
```

Comment:
1) `acl_token` seems not handled.
 
```
            headers = {}
            if acl_token:
                headers['X-Consul-Token'] = acl_token
```

2) Those line are probably not needed anymore
```
            clientcertfile = instance.get('client_cert_file', self.init_config.get('client_cert_file', False))
            privatekeyfile = instance.get('private_key_file', self.init_config.get('private_key_file', False))
```

3) The 3 self.http() calls can probably be reduced to one: